packages/rrweb/src/record/observers/canvas/canvas-manager.ts

```diff
@@ -121,14 +135,10 @@ export class CanvasManager implements CanvasManagerInterface {
         );
       if (recordCanvas && typeof sampling === 'number')
         this.initCanvasFPSObserver(
-          sampling,
           win,
           blockClass,
           blockSelector,
           unblockSelector,
-          {
-            dataURLOptions,
-          },
```

Comment:
I would leave these here so that this code doesn't diverge *too* much from upstream. Instead we can access `this.options` in `snapshot()` to pass to `manualSnapshot()`